+hw/+ptb/Window.m

```diff
@@ -385,6 +386,28 @@ function fillRect(obj, colour, rect)
     end
 
     function [nx, ny] = drawText(obj, text, x, y, colour, vSpacing, wrapAt)
+    % Calls PTB's `DrawFormattedText` to display text to the PTB Window
+    %
+    % Inputs:
+    %   `text`: the text to be displaye
+    %   `x`: a number, in pixels, of the x-position of the left border of 
+    %   the text
+    %   `y`: a number, in pixels, of the y-position of the top border of 
+    %   the text, in pixels
+    %   `colour`: an [r g b] or [r g b a] numeric vector for the color of 
+    %   the text
+    %   `vSpacing`: a number for the spacing between lines of text
+    %   `wrapAt`: a number for the max number of characters in a line of text
+    %
+    % Outputs:
+    %   `nx`: 
+    %   `ny`:
+    %
+    % Examples:
```

Comment:
Please remove this or add a TODO (or add examples)

Reply:
yup, this goes along with my comment in the initial PR where I say:

> I also want to improve the doc for hw.ptb.Window
_if_ you approve of this PR's general approach to solving #4, #128, #164 (I didn't want to waste time on documentation if you didn't approve of the code approach =D)

Reply:
Yeah looks good to me

Reply:
(Documenting existing code is never a waste of time! :)